### PR TITLE
Chore/eslint quotes rule

### PR DESCRIPTION
**Pull request by everdimension**

### CODE REVIEW

File: src/ui/pages/ConnectedSites/ConnectedSite/ConnectedSiteDialog.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ function ConnectedSitesPageLink() {
       kind="neutral"
       size={48}
       as={UnstyledLink}
-      to={`/connected-sites`}
+      to={'/connected-sites'}
```
Comment (zerts) [inline src/ui/pages/ConnectedSites/ConnectedSite/ConnectedSiteDialog.tsx:32]: We can remove braces here

File: src/ui/components/URLBar/URLBar.tsx [hunk 2]
```diff
@@ -104,7 +104,7 @@ export function URLBar() {
             <>
               <BackButton
                 onClick={() => navigate(getBackOrHome() as number)}
-                title={`Press "backspace" to navigate back`}
+                title={'Press "backspace" to navigate back'}
```
Comment (zerts) [inline src/ui/components/URLBar/URLBar.tsx:107]: We can remove braces here

File: src/ui/pages/Feed/Ability/AbilityPage.tsx [hunk 3]
```diff
@@ -120,7 +120,7 @@ export function AbilityPage() {
             ['--button-background' as string]: 'var(--white)',
           }}
           onClick={() => navigate(getBackOrHome() as number)}
-          title={`Press "backspace" to navigate back`}
+          title={'Press "backspace" to navigate back'}
```
Comment (zerts) [inline src/ui/pages/Feed/Ability/AbilityPage.tsx:123]: We can remove braces here

File: src/ui/components/URLBar/URLBar.tsx (hunk 2 above)
Comment (everdimension) [reply src/ui/components/URLBar/URLBar.tsx:107]: But there are double quotes in the message

Comment (zerts) [reply src/ui/components/URLBar/URLBar.tsx:107]: Oh, agree

**Review by zerts — APPROVED** [review]

**Pull request merged**